### DISCUSSION TITLE
Panel focus and right-click

### DISCUSSION THREAD

**Discussion by iljarogoff** [doublecmd/doublecmd#1814] [General]
Hi,
I'm using DC on manjaro linux with kde6 plasma. Everything works fine except one thing... and I have no clue if it can be fixed:
When marking files and/or folders with right-click mouse, the related panel does not become active (focus) if it is not focussed yet. You always need to left-click on the panel to set focus on it.
Already deleted lot of stuff because of this... i marked on the right panel some folders to delete, but oversaw that the focus still was on left panel. Deleting with F8 deleted on the left panel instead on right panel...
Is there a way to set focus with right-click?

**Comment by alexx2000** [comment]
Hi, copy here Double Commander version (Menu->Help->About->Copy to clipboard).

**Comment by iljarogoff** [comment]
Double Commander
Version: 1.1.17 gamma
Revision: 548
Commit: f4189f213
Build date: 2024/08/19
Lazarus: 3.4.0.0
FPC: 3.2.2
Platform: x86_64-Linux-qt6
OS version: Manjaro Linux

Widgetset library: Qt 6.7.2, libQt6Pas 6.2.1

**Comment by alexx2000** [comment]
Fixed.

**Reply by iljarogoff** [reply]
That was quick. I hope manjaro gets the fix asap :)

Edit: Tried it with the AppImage, works like a charm. Thanks a lot!
